application.py

```diff
@@ -1,53 +1,52 @@
 # Copyright 2013. Amazon Web Services, Inc. All Rights Reserved.
-# 
+#
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-# 
+#
 #     http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-import sys
 import json
+import logging
+import os
 
+import boto3
 import flask
-from flask import request, Response
-
-from boto import dynamodb2
-from boto.dynamodb2.table import Table
-from boto.dynamodb2.items import Item
-from boto.dynamodb2.exceptions import ConditionalCheckFailedException
-from boto import sns
+from flask import Response, request
 
 # Default config vals
-THEME = 'default' if os.environ.get('THEME') is None else os.environ.get('THEME')
-FLASK_DEBUG = 'false' if os.environ.get('FLASK_DEBUG') is None else os.environ.get('FLASK_DEBUG')
+THEME = 'default' if os.environ.get(
+    'THEME') is None else os.environ.get('THEME')
+FLASK_DEBUG = 'false' if os.environ.get(
+    'FLASK_DEBUG') is None else os.environ.get('FLASK_DEBUG')
 
 # Create the Flask app
 application = flask.Flask(__name__)
 
 # Load config values specified above
-application.config.from_object(__name__)
+application.config.from_object("default_config")
```

Comment:
We should be consistent about using single quotes and double quotes

Reply:
Yes. Will update this part